mesonbuild/modules/gnome.py

```diff
@@ -377,6 +380,10 @@ def _get_dependencies_flags(self, deps, state, depends=None, include_rpath=False
 
         return cflags, ldflags, gi_includes
 
+    @permittedKwargs(set(['sources', 'nsversion', 'namespace', 'symbol_prefix', 'identifier_prefix',
+                         'export_packagse', 'includes', 'dependencies', 'link_with', 'include_directories',
+                         'install', 'install_dir_gir', 'install_dir_typelib', 'extra_args',
+                         'packages']))
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->